### ISSUE TITLE
`Scene.display()` should open the visualization in a minaml IPython notebook

### ISSUE THREAD

**Issue by moorepants** [pydy/pydy#186]
Right now we do not have the interactivity, i.e. updating the simulation, in when the visualization is displayed outside of the notebook, i.e. calling `Scene.display_ipython`. We should, instead, open a minimal IPython notebook (single cell, no header bars, etc) to run the visualization so we can have the widget functionality whether you are in a notebook cell among many in a notebook or just open the visualization standalone.

**Comment by moorepants** [comment]
It doesn't seem like you can reliably know if code was executed in the notebook or not, see: http://stackoverflow.com/questions/15411967/how-can-i-check-if-code-is-executed-in-the-ipython-notebook

For correct IPython notebook functionaliy we should use the IPython `display()` function and/or the `_html()` methods. Then Scene can be displayed `display(Scene)` or simply creating a Scene object will create the simulation cell.

**Comment by tarzzz** [comment]
Inside the `display_ipython` we are making multiple calls to IPython's `display` to display all the components (simulation buttons etc.). `display_ipython` is more like an abstraction to `display(Scene)` since it loads all the relevant objects in a single call. 

One way to achieve what you intend is to treat (and refractor) our Scene object as a Custom IPython widget, so that IPython display call can work on it. Would be nice to use `display(Scene)`, but might not be robust, since IPython guys change their codes quite frequently.

**Comment by moorepants** [comment]
Yes, `display(Scene)` seems like the "right" way to do things, at least in IPython's eyes.
